<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionEditionWidgetBase</class>
 <widget class="QWidget" name="OptionEditionWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QFrame" name="Frame3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32567</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item>
       <widget class="QLabel" name="title_label">
        <property name="font">
         <font>
          <family>Nimbus Sans l [Urw]</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Option Management</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="Spacer3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>181</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QGroupBox" name="type_groupbox">
       <property name="maximumSize">
        <size>
         <width>32767</width>
         <height>240</height>
        </size>
       </property>
       <property name="title">
        <string>Option Type</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLineEdit" name="type_lineedit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="type_combobox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_type_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Add option type</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image0</normaloff>image0</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QTreeWidget" name="type_listview">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>32767</width>
              <height>32767</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="up_type_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>100</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Upload an option type</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoRepeat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="down_type_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>100</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Download a type of option</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoRepeat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QPushButton" name="del_type_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Delete an option type</string>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image3</normaloff>image3</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="type_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>241</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="option_groupbox">
       <property name="maximumSize">
        <size>
         <width>32767</width>
         <height>240</height>
        </size>
       </property>
       <property name="title">
        <string>Option</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLineEdit" name="option_lineedit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="option_combobox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add_option_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Add option to selected type</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image0</normaloff>image0</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QTreeWidget" name="option_listview">
            <property name="maximumSize">
             <size>
              <width>32767</width>
              <height>32767</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="up_option_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>32767</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Upload an option</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoRepeat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="down_option_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>32767</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Download an option</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoRepeat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QPushButton" name="del_option_button">
            <property name="toolTip">
             <string>Delete Option</string>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>image3</normaloff>image3</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="option_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>241</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer31_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="group_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Set value to all products</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QGroupBox" name="buttonGroup1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="exclusive" stdset="0">
             <bool>true</bool>
            </property>
            <property name="radioButtonExclusive" stdset="0">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout">
             <item>
              <widget class="QPushButton" name="plus_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Establish positive variant on the chosen option</string>
               </property>
               <property name="text">
                <string>+ (Positive)</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="minus_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Establish negative variant on the chosen option</string>
               </property>
               <property name="text">
                <string>- (negativ)</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="spacer12">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>61</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="box">
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>250</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QLabel" name="advertise_label">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32767</width>
              <height>3000</height>
             </size>
            </property>
            <property name="text">
             <string>If you want to apply the option to all products, click apply. Attention, if you add a value to this option, all the products will add to their original price, the value entered. If you want to set a price for each product, go to the products section.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer28">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <spacer name="spacer31_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="box_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Set the variation on price and option to all products in the database.</string>
              </property>
              <property name="text">
               <string>Apply</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>image4</normaloff>image4</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>41</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="spacer29">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>130</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer11">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <spacer name="Spacer1_2_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>390</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
